src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -84,4 +84,23 @@ export class InputActionDialog {
   close() {
     this.modal.close();
   }
+
+  checkIfJenkinsIsUp() {
+    let url = 'https://jenkins.openshift.io';
```

Comment:
A new environment needs to be added to F8UI (the Jenkins URL)

Reply:
I wanted to inject an environment variable like https://github.com/fabric8-ui/fabric8-ui/blob/master/src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts#L5-L26
Where and how do I do this

Reply:
@kishansagathiya I think @edewit or @joshuawilson may know this better than me :wink: